wallet/wallet.go

```diff
@@ -456,11 +459,17 @@ func (w *Wallet) SynchronizeRPC(chainClient *chain.RPCClient) {
 			"and missed tickets. Error: ", err.Error())
 	}
 
-	if w.StakeMiningEnabled {
-		log.Infof("Stake mining is enabled. Votebits: %v, minimum wallet "+
-			"balance %v", w.VoteBits, w.BalanceToMaintain().ToCoin())
-		log.Infof("PLEASE ENSURE YOUR WALLET REMAINS UNLOCKED SO IT MAY " +
-			"VOTE ON BLOCKS AND RECEIVE STAKE REWARDS")
+	ticketPurchasingEnabled := w.TicketPurchasingEnabled()
+	if ticketPurchasingEnabled {
+		log.Infof("Wallet ticket purchasing enabled: vote bits = %v, "+
+			"balance to maintain = %v", w.VoteBits, w.BalanceToMaintain)
```

Comment:
BalanceToMaintain is an Amount which is already a fmt.Stringer.  This is technically a change in behavior, but it's just logs, and the only real difference will be the addition of "Coin" in the message.